src/main/java/org/embulk/output/GcsOutputPlugin.java

```diff
@@ -419,6 +419,12 @@ private String getLocalMd5hash(String filePath) throws IOException
         }
     }
 
+    private static String generateRemotePath(String pathPrefix, String sequenceFormat, int taskIndex, int fileIndex, String pathSuffix)
+    {
+        String path = pathPrefix + String.format(sequenceFormat, taskIndex, fileIndex) + pathSuffix;
+        return path.replaceFirst("\\.*/*", "");
```

Comment:
Got it. I was concerned about `.foobar` in addition, but it is not accepted by Google.

Then, can you add a comment with:
* What this normalization intends
* Link(s) to GCS documents about its filenames
* Why this normalization works for GCS, in short